openassessment/assessment/api/staff.py

```diff
@@ -54,18 +54,27 @@ def assessment_is_finished(submission_uuid, requirements):
     Returns:
         True if a staff assessment has been completed for this submission or if not required.
     """
-    if requirements and requirements.get('staff', {}).get('required', False):
```

Comment:
Are requirements no longer separated out into staff-specific requirements?  Or just in this method?  Should the param be staff_requirements?

Note: I'm commenting as I read, so this may become clear later.